M2/Macaulay2/packages/ReesAlgebra.m2

```diff
@@ -209,26 +207,50 @@ isLinearType(Module, RingElement):= o-> (N,a)->(
      J := ideal((vars S) * P);
      ((gens I) % J) == 0)
 
-normalCone = method(TypicalValue => Ring, 
-    	    Options => {
+-- normalCone = method(TypicalValue => Ring, 
+--     	    Options => {
+-- 	  DegreeLimit => {},
+-- 	  BasisElementLimit => infinity,
+-- 	  PairLimit => infinity,
+-- 	  MinimalGenerators => true,
+-- 	  Strategy => null,
+-- 	  Variable => "w"
+-- 	  }
+-- )
+-- normalCone(Ideal) := o -> I -> (
+--      RI := reesAlgebra(I,o);
+--      RI/promote(I,RI)
+--      )
+
+-- normalCone(Ideal, RingElement) := o -> (I,a) -> (
+--      RI := reesAlgebra(I,a,o);
+--      RI/promote(I,RI)     
+--      )
```

Comment:
Is there a reason this section is duplicated here? It's all just below, or am I missing something?

Reply:
commented out code removed, thanks for noticing!